_includes/v22.2/ui/insights.md

```diff
@@ -0,0 +1,224 @@
+## Workload Insights tab
+
+The **Workload Insights** tab displays insights related to transaction and statement executions.
+
+### Transaction Executions view
+
+{% if page.cloud != true -%}
+To display this view, click **Insights** in the left-hand navigation of the DB Console and select **Workload Insights > Transaction Executions**.
+{% endif -%}
+{% if page.cloud == true -%}
+{{site.data.alerts.callout_info}}
+The **Transaction Executions** view is currently unavailable for {{ site.data.products.serverless }} clusters.
+{{site.data.alerts.end}}
+
+To display this view, click **Insights** in the left-hand navigation of the Cloud Console and select **Workload Insights > Transaction Executions**.
+{% endif -%}
+
+The **Transaction Executions** view provides an overview of all transaction executions that have been flagged with insights.
```

Comment:
"transaction" could be a link to https://www.cockroachlabs.com/docs/v22.2/transactions.html for general background info